extensions/resource-deployment/src/ui/toolsAndEulaSettingsPage.ts

```diff
@@ -364,8 +387,6 @@ export class ToolsAndEulaPage extends ResourceTypePage {
 	private setUiControlsEnabled(enable: boolean): void {
 		this._agreementContainer.enabled = enable;
 		this._optionsContainer.enabled = enable;
-		this.wizard.wizardObject.cancelButton.enabled = enable;
```

Comment:
are we allowing cancel button to be clickable when the tool is being installed? If so, we should revert this change as   
we do not have a way of cancelling already started tool installation. Install once started has to be allowed to proceed to completion.